src/sender.ts

```diff
@@ -123,7 +127,23 @@ export class Sender {
   }
 
   private reSend() {
-    if (this.#latestPacketOptions) this.send(this.#latestPacketOptions);
+    if (this.#latestPacketOptions) {
+      this.send(this.#latestPacketOptions)
+        .then(() => {
+          this.updateResendStatus(true);
+        })
+        .catch((err) => {
+          this.updateResendStatus(false);
+          this.emit('error', err);
+        });
+    }
+  }
+
+  private updateResendStatus(success: boolean) {
+    if (success !== this.resendStatus) {
+      this.resendStatus = success;
+      this.emit('changedResendStatus', success);
```

Comment:
could you please declare these 2 new events using an `EventMap`, just like we do for `Receiver`?

example here:

https://github.com/k-yle/sACN/blob/b7d8017a7b03f507db124bc28d1633b80a423d1e/src/receiver.ts#L23-L35

Reply:
fixed in edd2ec5d909c1ddb25049e97b2145a1a3daa305c